MDM: replace deprecated HTML tag with MarkDown (#9474)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/cmpolicyenterprise-csp.md

</tbody>
</table>

<a href="" id="type"></a>**Type**
Specifies the type of connection being referenced. The following list describes the available connection types:

- `CMST_CONNECTION_NAME` – A connection specified by name.

- `CMST_CONNECTION_TYPE` – Any connection of a specified type.

> - `CMST_CONNECTION_NETWORK_TYPE` – Any connection of a specified device type.

Reviewer: ```suggestion
- `CMST_CONNECTION_NETWORK_TYPE` – Any connection of the specified network type.
```

> - `CMST_CONNECTION_DEVICE_TYPE` – Any connection of the specified network type.

Reviewer: ```suggestion
- `CMST_CONNECTION_DEVICE_TYPE` – Any connection of a specified device type.
```
